stories/Drawer.stories.svelte

```diff
@@ -12,25 +12,23 @@
 			{
 				url: './',
         icon: 'account_circle',
-				label: 'see our components in storybook'
+				label: 'reload'
```

Comment:
Looks like this file has mixed tabs and spaces, making it slightly harder to read.

Reply:
reformatting cleared up the weirdness. I wonder if there is a prettier config we could all agree on for js.

Reply:
Sounds worth trying 👍